### ISSUE TITLE
Обнаружена ошибка кодирования, она должна быть исправлена программистом: WebEx exception 030098 when creating new user.  Подробнее об этой ошибке

### ISSUE THREAD

**Issue by Aniraz25** [merrill-oakland/moodle-mod_webexactivity#28]

**Comment by Aniraz25** [comment]
Обнаружена ошибка кодирования, она должна быть исправлена ​​программистом: исключение WebEx 030098 при создании нового пользователя.

**Comment by ericmerrill** [comment]
That error means you are using a unified site, and my plugin can't create the user. The way around this it to make sure that the user in Moodle who is trying to make a meeting has the same username and email as an already existing user in WebEx.

**Comment by Aniraz25** [comment]
у нас пользователи в moodl входят по AD, имя и адрес электронной почты в webex передаются по AD. Что в данном случае можно попробовать?//we have users logging into moodl via AD, the name and email address in webex are transmitted via AD. What can you try in this case?

> Эта ошибка означает, что вы используете единый сайт, и мой плагин не может создать пользователя. Чтобы обойти это, необходимо убедиться, что пользователь в Moodle, который пытается провести собрание, имеет то же имя пользователя и адрес электронной почты, что и уже существующий пользователь в WebEx.

**Comment by Cudmohammadfayaz** [comment]
> That error means you are using a unified site, and my plugin can't create the user. The way around this it to make sure that the user in Moodle who is trying to make a meeting has the same username and email as an already existing user in WebEx.

Hello.  I'm having the same issue.  It was working for the last 2 years without any issues.  It seems Cisco is pushing all accounts to unified and there is no way to make go back. 
Which table is this user list populated in and if we create a user entry in this table, would it stop the code from sending a createuser request?

**Comment by ericmerrill** [comment]
The user table is mdl_webexactivity_user. If you can get all the correct info to populate that, I think it would work.

**Comment by Cudmohammadfayaz** [comment]
Updating the table with necessary fields worked.
Thank you.


Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by Cudmohammadfayaz** [comment]
CI-enabled webex sites seems to be very common now as institutions are integrating webex with AD.  I think a good enhancement would be to allow mdl_webexactivity_user table to be updated from the front-end in Moodle admin settings.  The table is already exposed in the interface.  The missing feature is allow insert/Modification.

**Comment by ericmerrill** [comment]
Yeah, I'm glad this is working, but I actually no longer have access to webex (Oakland stopped using it), and I moved to a new job, so I don't have much time to work on it.
I'm happy to keep it loosely working, and integrate contributions people make, but it's unlikely that I will be able to develop new features anytime soon 😕

**Comment by metheoryt** [comment]
Hi! I've just installed plugin and trying to test it. I'm getting this exception when trying to create meeting.
I've tried to create meeting under a user with same username and email as a webex organizer, but got the same error.
Do you have any idea on what could be wrong?

**Comment by ericmerrill** [comment]
Unfortunately, no, I'm not sure what might be happening - in the past it has always been related to the username/password issue.
